xml/art-obs-beginners-guide.xml

```diff
@@ -118,19 +118,10 @@ https://www.suse.com/communities/blog/suse-studio-integration/
   </sect2>
   <sect2 xml:id="sec.obsbg.buildhost">
    <title>Build Hosts and Packages</title>
-  <remark>
-   Maybe "You can communicate with OBS either through..."? Hesitant to apply
-   this because while more correct it sounds about as weird as the original.
-   - sknorr, 2017-08-15
-  </remark>
   <para>
-   The &obs; is accessible either through a Web interface or a command
-   line interface. Both access the OBS API which controls the database,
-   <remark>
-    "Controls" seems like the wrong word. Not sure about a better on right
-    now. - sknorr, 2017-08-15
-   </remark>
-   <remark>toms 2017-08-16: maybe "communicates" instead of "controls"?</remark>
+   The &obs; is accessible either through a Web interface or a command line
+   interface (&osccmd;). Both are clients for the &obsa; API which provides
+   access to the database,
```

Comment:
Ok, in this case, I would like to remove this sentence ("Both are...") as it seems an implementation detail.